docs/checkit.js

```diff
@@ -865,10 +834,32 @@ Calendar.prototype.fillCalendar = function(monthObjectsArray) {
     monthObjectsArray.forEach (function(monthObj) {
         monthObj.fillMonthDiv();
         monthObj.removeEmptyWeeks();
-        monthObj.attachClickHandler();
-        monthObj.generateCheckmarks();
     });
 };
 
 
+Calendar.prototype.generateCheckmarks = function() {
+    // Removes the hidden class between the children of the div class="cell" 
+    // of the cells whose indices are in the monthState.checkedDays
+    // object.
+    
+    //checkedDays is an object that contains a date that points to 1 or 0
+    var self = this;
+    if (self.state.checkedDays === undefined) {
+        console.log("No days are checked");
+        return;
+    }
+    
+    $('#calendarDiv').find('.cell').each( function() {
```

Comment:
If this function took this element as a parameter, you could use it to initialize single months or the whole calendar as needed.